src/Mutation/RootMutation.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atoolo\GraphQL\Search\Mutation;
+
+use Overblog\GraphQLBundle\Annotation as GQL;
+
+/**
+ * TODO: This should be a general Root-Mutation for all GraphQL extensions and
+ * should not be in this search bundle. However, there is no general Atoolo
+ * GraphQL bundle yet and is therefore stored here for the time being.
+ */
```

Comment:
I would leave it in here for now. I can imagine that we still need a general atoolo-graphql-bundle if we create another atoolo-graphql-xxx-bundle. So that we have a place where, for example, the general types are located. At the moment it doesn't matter. So I would leave it as it is.

I don't think it's a good idea to put it in the application.

Reply:
There's probably a good reason why `Overblog\GraphQL` did not include these `Root` classes either; It _seems_ like they want these to be defined in the application.  
As you might pick up from my wording, I have no idea why it is that way. I'd just like to make sure we're not working against the intended design here.

Reply:
But if it does not matter where those are defined than why not in the bundle? they're empty classes, it can't be a bloat issue.  

It probably does not matter. We can change it once it turns out to be a problem.